gurubu-client/src/app/styles/variables.scss

```diff
@@ -53,13 +56,15 @@ $red-700: #dc2626;
 
 
 // font size
-$font-size-header-1: 50px;
```

Comment:
If you change the variable values, it will affect all of the pages. In other pages maybe 50px needed did you change all of the pages variable ?

Reply:
if you need font-size-header-1 to be more than that value in your development than you should change other usaged in the code with font-size-header-2

Reply:
yeap, all variables updated. fyi

Reply:
https://gurubu-46p281eh3-trendyol-gurubu-team.vercel.app/create/room on this page Gurubu text should be 26px but it seems like 42px can you check ?

Reply:
it must be 24px, it looks like that in figma so i changed the font-size and font-weight according to the figma.